triggers/service/BUILD.bazel

```diff
@@ -72,8 +73,12 @@ da_scala_binary(
 da_scala_test(
     name = "tests",
     srcs = glob(["src/test/scala/**/*.scala"]),
-    data = [":test-model.dar"],
+    data = [
+        ":test-model.dar",
+        "@toxiproxy_nix//:bin/toxiproxy-cmd",
+    ],
     resources = glob(["src/test/resources/**/*"]),
+    tags = ["exclusive"],  # We require hard-coded ports for this test.
```

Comment:
Can we split up the test suite into tests that use Toxiproxy and those that don't? Then we can still have some tests running on windows and reduce the amount of testing that requires `exclusive`.

Reply:
Maybe. It's possible to run toxiproxy on Windows so I think it's better to invest our efforts in that.

Reply:
What is the issue you saw running the tests on Windows?